### PR TITLE
Renamed accent button style name to AccentButtonStyle

### LINKED ISSUE #9059: [Theming] DefaultAccentButtonStyle should be AccentButtonStyle
### Description

The new `DefaultAccentButtonStyle` allows you to turn a button into a colored button using the system's accent color.

This is based on the WinUI style called `AccentButtonStyle` (sans the `Default`)

### Reproduction Steps

N/a

### Expected behavior

Use the `AccentButtonStyle` style name

WinUI has the following Button styles:
`DefaultButtonStyle`
`AccentButtonStyle`

### Actual behavior

I have to use the `DefaultAccentButtonStyle`

### Regression?

_No response_

### Known Workarounds

_No response_

### Impact

_No response_

### Configuration

_No response_

### Other information

_No response_

### PR DESCRIPTION
**Pull request by siagupta0202**
Fixes #9059

## Description
Renamed accent button style name to AccentButtonStyle to match WinUI

###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/wpf/pull/9494)

### CODE REVIEW

**Review by dipeshmsft — DISMISSED** [review]

File: src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Fluent/Themes/Fluent.HC.xaml [hunk 1]
```diff
@@ -3067,7 +3067,7 @@
     <Setter Property="MinHeight" Value="{DynamicResource TextControlThemeMinHeight}" />
     <Setter Property="MinWidth" Value="{DynamicResource TextControlThemeMinWidth}" />
     <Setter Property="Padding" Value="{DynamicResource TextControlThemePadding}" />
-    <Setter Property="PasswordChar" Value="●" />
+    <Setter Property="PasswordChar" Value="â—" />
```
Comment (lindexi) [inline src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Fluent/Themes/Fluent.HC.xaml:3070]: What is this encoding?

Comment (miloush) [reply src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Fluent/Themes/Fluent.HC.xaml:3070]: That is clearly UTF-8 read as CP 1252. BOM might help.

Comment (dipeshmsft) [reply src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Fluent/Themes/Fluent.HC.xaml]: Reverted this.

**Review by dipeshmsft — APPROVED** [review]

File: src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Fluent/Themes/Fluent.HC.xaml (hunk 1 above)
Comment (dipeshmsft) [reply src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Fluent/Themes/Fluent.HC.xaml]: Reverted this. Resolving conversation

**Pull request merged**